test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetInstallCommandTest.cs

```diff
@@ -484,8 +484,7 @@ public void InstallCommand_FromPackagesConfigFileFailsVerifyCode()
 
                 // Assert
                 Assert.Equal(1, r.Item1);
-                r.AllOutput.Should().NotContain("NU1000");
-                r.Errors.Should().Contain("Unable to find version");
```

Comment:
Previously there was error but still not error code but checking NU1000 not contained was arbitrary, instead new implementation emitting NU1000 much correct (not exact correct) behavior.